src/main/java/training/subsystems/elbow/ElbowSimulation.java

```diff
@@ -35,6 +36,11 @@ private ElbowSimulation() {
 		elbowSimulation.applyConfiguration(config);
 	}
 
+	public void goToAngle(Rotation2d targetAngle) {
+		pos = new PositionVoltage(targetAngle.getRotations());
+		elbowSimulation.setControl(pos);
```

Comment:
```suggestion
		elbowSimulation.setControl(pos.withPosition(targetAngle.getRotations()));
```
consider doing this instead